src/data/actions/loginForm.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+import qs from 'query-string';
+
+import {
+  LOGIN_FAILURE,
+  LOGIN_REQUEST,
+  LOGIN_SUCCESS,
+} from '../constants/loginForm';
+
+const setLoginPending = loading => ({
+  type: LOGIN_REQUEST,
+  payload: { loading },
+});
+
+const setLoginSuccess = isLoginSuccess => ({
+  type: LOGIN_SUCCESS,
+  payload: { isLoginSuccess },
+});
```

Comment:
You don't need to pass the `isLoginSuccess` variable into this function as `isLoginSuccess: true` can be set within the reducer for the `LOGIN_SUCCESS` action.